x-pack/plugins/actions/server/config.ts

```diff
@@ -37,6 +38,8 @@ export const configSchema = schema.object({
   proxyHeaders: schema.maybe(schema.recordOf(schema.string(), schema.string())),
   proxyRejectUnauthorizedCertificates: schema.boolean({ defaultValue: true }),
   rejectUnauthorized: schema.boolean({ defaultValue: true }),
+  maxResponseContentLength: schema.number({ defaultValue: 1000000 }), // 1MB
+  responseTimeout: schema.string({ validate: validateDurationSchema, defaultValue: '60s' }),
```

Comment:
You may want to use the built-in `schema.duration()`:
```suggestion
  responseTimeout: schema.duration({ defaultValue: '60s' }),
```